language/move-prover/bytecode-to-boogie/src/translator.rs

```diff
@@ -617,15 +617,33 @@ impl<'a> ModuleTranslator<'a> {
                 "call tmp := LdFalse();".to_string(),
                 format!("m := UpdateLocal(m, old_size + {}, tmp);", idx),
             ],
-            LdU8(_, _) => unimplemented!(),
-            LdU64(idx, num) => vec![
-                format!("call tmp := LdConst({});", num),
-                format!("m := UpdateLocal(m, old_size + {}, tmp);", idx),
+            LdU8(idx, num) =>
```

Comment:
LdU8(idx, num) | LdU64(idx, num) | LdU128(idx, num) => ...

?

Reply:
I tried this, but it won't let me do it because the type of `num` is different in each variant.